src/ssl_stubs.c

```diff
@@ -256,12 +256,18 @@ CAMLprim value ocaml_ssl_init(value use_threads)
 
 CAMLprim value ocaml_ssl_get_error_string(value unit)
 {
-  CAMLparam1(unit);
+  CAMLparam0();
```

Comment:
this should be `CAMLparam1(unit)`, according to examples in the manual https://v2.ocaml.org/manual/intfc.html

Reply:
Current binding of open_ssl were using the possibility to not register parameter which are known to be integer, so I did like

```
CAMLprim value ocaml_ssl_read(value socket, value buffer, value start, value length)
{
  CAMLparam2(socket, buffer);
```

This does not seem officialy documented but already used by open_ssl?